Java/src/main/java/Planes/ExperimentalPlane.java

```diff
@@ -3,14 +3,12 @@
 import models.ClassificationLevel;
 import models.ExperimentalTypes;
 
```

Comment:
fix package name of this file. according to Java Convention packages should be written in lowercase:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html